design/object-bucket/object-bucket.md

```diff
@@ -6,29 +6,26 @@ Enable Rook-Ceph Object users the ability to request buckets in an object store
 
 ### Overview
 
-  Rook-Ceph has developed Custom Resource Definitions (CRDs) to support the dynamic provisioning of object stores and users (CephObjectStores and CephObjectStoreUsers, respectively).  This proposal introduces the next logical step in this design: the addition a controller for handling dynamic bucket creation requests represented the ObjectBucketClaim and ObjectBucket APIs.  The intent is to round out the Rook-Ceph experience for Kubernetes users by providing a single control plane for the managing of Ceph Object components.  The ObjectBucketClaim and ObjectBucket APIs will represent generic S3 bucket creation requests and can be implemented by any object store provider.
-  
-  Rook-Ceph will be the first operator to utilize these new APIs.  The APIs themselves will be temporarily implemented in the Rook project as a discrete package and later be submitted for acceptance into a repository with broader Kubernetes support (for instance, [Kubernetes Enhancements](https://github.com/kubernetes/enhancements)).  By implementing dynamic bucket creation in Rook-Ceph first, the object bucket APIs will be able to present a stronger case for adoption when submitted to the to-be-determined project.
+Rook-Ceph has developed Custom Resource Definitions (CRDs) to support the dynamic provisioning of object stores and users (CephObjectStores and CephObjectStoreUsers, respectively).  This proposal introduces the next logical step in this design: the addition a controller for handling dynamic bucket creation requests via the generalized ObjectBucketClaim and ObjectBucket APIs.  These APIs are defined in an external project with the intent of working to garner broader community support.  The intent is to round out the Rook-Ceph experience for Kubernetes users by providing a single control plane for the managing of Ceph Object components.  The ObjectBucketClaim and ObjectBucket APIs represent generic S3 bucket creation requests to enable the provisioning of buckets by most object store providers.
 
 ## Glossary
 
-- Admin: A Kubernetes cluster administrator with RBAC permissions to instantiate and control access to cluster storage. The admin will use existing Rook-Ceph CRDs to create Ceph Object Stores as well as StorageClasses to make them accessible to users.  
+- Admin: A Kubernetes cluster administrator with RBAC permissions to instantiate and control access to cluster storage. The admin will use provision object stores via CephObjectStore custom resources and create StorageClasses to enable user access.  
 - User: A Kubernetes cluster user with limited permissions, typically confined to CRUD operations on Kubernetes objects within a single namespace.  The user will request Ceph Object buckets by instantiating ObjectBucketClaims.
```

Comment:
_The user will request Ceph Object buckets by instantiating ..._
In keeping with the generic bucket abstraction for the user, should "Ceph" be in this sentence?

Reply:
This design is for the implementation of a Ceph controller.